code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1012,6 +1012,11 @@
 	riding_datum.handle_vehicle_layer()
 	. = ..(target, force, check_loc)
 
+	//Something went wrong with buckling, unbuckle!
+	if(!.)
+		SEND_SIGNAL(src, COMSIG_MOVABLE_UNBUCKLE, target, force)
```

Comment:
are you sure you are supposed to send the signal even though we have never actually buckled the target?

Reply:
What the COMSIG_MOVABLE_UNBUCKLE does with that component is removing the inhands and restoring position, but I guess there could be multiple things listening to the same signal so yeah, better do it 'by hand' I think.